### DISCUSSION TITLE
Does free credit have an expire delay?

### DISCUSSION THREAD

**Discussion by jdbruxelles** [deepgram/community#112] [General help] [answered]
Hello, I am currently benefiting from the free offer which is very interesting to test the service before investing in it. Just, I would like to know if the credits offered have an unlimited life span or if I can use them as I wish.

**Comment by briancbarrow** [comment, marked as answer]
Hi @jdbruxelles the free credits expire after 1 year from signup.

**Reply by jdbruxelles** [reply]
Ok, thank you, @briancbarrow.
